docker-compose.yml

```diff
@@ -1,72 +1,52 @@
-
-version: '2'
+version: '3'
 
 services:
   php56:
     build:
-      context: .
-      dockerfile: docker/Dockerfile-php56
+      context: docker
+      dockerfile: Dockerfile-php56
     volumes:
-      - './spec/:/usr/src/gaufrette/spec/'
-      - './src/:/usr/src/gaufrette/src/'
-      - './tests/:/usr/src/gaufrette/tests/'
+      - './:/usr/src/gaufrette:rw'
```

Comment:
If you do this we won't be able to run multiple php versions at the same time. Currently only `spec`, `src` and `tests` are bind-mounded to not mount `vendor`.

Reply:
That's why the entrypoint script is here.

It will update the vendor when running the tests so we always have dependencies that are compatible with the runned PHP version.

With this, running the tests is just a one command line : 
```sh
bin/tests-all
```

Reply:
Yeah but it means that we can't run different php versions in parallel during development and we've to reinstall dependencies again and again whereas they don't change. Tests should be fast to execute and no extra noise should be added.